### DISCUSSION TITLE
Panache REST resources - Related Entities (lazy loading)

### DISCUSSION THREAD

**Discussion by trm217** [quarkusio/quarkus#41691] [Q&A]
Given these tutorials
- https://quarkus.io/guides/hibernate-reactive-panache
- https://quarkus.io/guides/rest-data-panache

As soon as related entities are used, one comes across lazy-load errors all over the place.
There are no resources on what one has to do.
If you implement an endpoint yourself, you can go back to using `Muteny.fetch`, however with panache  rest resources, this isn't an option.

Are there any resources hidden somewhere on what can be done?

[a comment by an automated account is not shown]

**Comment by FroMage** [comment]
Hi,

I don't think REST Data Panache works well with relations, as you've observed. If you feel like contributing support for it, that'd be great, and we can help you :)
